packages/core/src/checkbox/checkbox.ts

```diff
@@ -8,6 +8,8 @@ import { event } from '../utils/decorators/index.js';
 import { FormControlMixin, HintMixin } from '../utils/mixins/index.js';
 import styles from './checkbox.scss.js';
 
+export type CheckboxSize = 'md' | 'lg';
```

Comment:
Just discussed this with @DanielleRameau : we probably need an `sm` size (eventually) as well.

Reply:
Will add that when it's relevant. Right now we also don't have the tokens yet